### PR TITLE
chore: clarify encode protobuf message need to be qualified by package path

### PR DESCRIPTION
**Pull request by xiangjinwu**
## Info

- **Description**

  - Give better example on `encode protobuf` for source & sink

- **Notes**

  - Did not update older v1.8, v1.7, ... unless there is an easier way to do so

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves [ Provide a link to the relevant doc issue here, if applicable. ]

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: docs/guides/create-sink-kafka.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ When creating an append-only Protobuf sink, the following options can be used fo
 
 |Field|Notes|
 |-----|-----|
-|message| Required. Message name of the main Message in the schema definition. . |
+|message| Required. Package qualified message name of the main Message in the schema definition. . |
```
Comment (ShanlanLi) [inline docs/guides/create-sink-kafka.md]:
```suggestion
|message| Required. Package qualified message name of the main Message in the schema definition.  |
```

Comment (ShanlanLi) [reply docs/guides/create-sink-kafka.md]: Here are two dots. Is it a typo?

File: versioned_docs/version-1.9/guides/create-sink-kafka.md [hunk 2]
```diff
@@ -130,7 +130,7 @@ When creating an append-only Protobuf sink, the following options can be used fo
 
 |Field|Notes|
 |-----|-----|
-|message| Required. Message name of the main Message in the schema definition. . |
+|message| Required. Package qualified message name of the main Message in the schema definition. . |
```
Comment (ShanlanLi) [inline versioned_docs/version-1.9/guides/create-sink-kafka.md]:
```suggestion
|message| Required. Package qualified message name of the main Message in the schema definition. |
```

**Review by ShanlanLi — APPROVED** [review]
LGTM, thanks.

File: docs/guides/create-sink-kafka.md (hunk 1 above)
Comment (xiangjinwu) [reply docs/guides/create-sink-kafka.md]: Not introduced by this PR but I also believe it was a typo

**Pull request merged**